DSharpPlus/BaseDiscordClient.cs

```diff
@@ -174,8 +174,20 @@ public virtual async Task InitializeAsync()
             }
         }
 
-        internal DiscordUser InternalGetCachedUser(ulong user_id) 
-            => this.UserCache.TryGetValue(user_id, out var user) ? user : new DiscordUser { Id = user_id, Discord = this };
+        internal DiscordUser GetCachedOrEmptyUserInternal(ulong user_id)
+        {
+            TryGetCachedUserInternal(user_id, out var user);
+            return user;
+        }
+
+        internal bool TryGetCachedUserInternal(ulong user_id, out DiscordUser user)
+        {
+            if (this.UserCache.TryGetValue(user_id, out user))
+                return true;
+
+            user =  new DiscordUser {Id = user_id, Discord = this};
```

Comment:
Just a nitpick: please update whitespace to be inline with the rest, i.e.: `new DiscordUser { Id = user_id, Discord = this }`

Reply:
I believe this is the only instance. Idk why ReSharper isn't picking up solution code style.